src/roomspace.c

```diff
@@ -834,29 +1049,134 @@ static void keeper_update_roomspace(struct RoomSpace *roomspace)
             roomspace->is_active = false;
             return;
         }
-        keeper_build_room(slab_subtile(roomspace->buildx, 0), slab_subtile(roomspace->buildy, 0),
```

Comment:
The alignment was correct before your change I think,... either that or on a single line.

Reply:
What do you mean?